src/iwp.jl

```diff
@@ -82,6 +82,20 @@ function _transition_matrix_1d(ndiff::Integer, dt::T, ::ReverseTaylor) where {T<
 end
 
 
+precond(ndiff::Integer, dt::Real, ::ReverseTaylor) =  @. sqrt(dt) * dt^(0:ndiff) / factorial(0:ndiff)
```

Comment:
Is there a particular reason for why this doesn't output a `Diagonal` directly? Then all the functiosns return matrices.

Reply:
We could certainly return a Diagonal matrix instead. But now, weekend.

Reply:
@nathanaelbosch fine now?